keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -237,7 +237,7 @@ def generate_image(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
-    def text_to_image_prompt_to_prompt(
+    def prompt_to_prompt(
```

Comment:
is there any way to take an input image instead of an input prompt?

Reply:
Hi @LukeWood, sorry for the delay.
No :( I believe the authors mention this in the paper (as added future work). 
Also, there is an issue in the original code repo mentioning [this](https://github.com/google/prompt-to-prompt/issues/8).